erigon-lib/downloader/webseed.go

```diff
@@ -212,10 +244,23 @@ func (d *WebSeeds) downloadTorrentFilesFromProviders(ctx context.Context, rootDi
 					d.logger.Log(d.verbosity, "[snapshots] got from webseed", "name", name, "err", err, "url", url)
 					continue
 				}
+				// Compute info-hash before creating the file
```

Comment:
I guess we have same logic inside `d.callTorrentHttpProvider` -> `validateTorrentBytes`

Reply:
Good point, thank you, I will simplify